test/unit.js

```diff
@@ -8,13 +8,31 @@ import './unit/slot/distribution';
 import './unit/slot/fallback-content';
 import './unit/slot/polyfill';
 import './unit/webcomponents/initialisation';
+import canPatchNativeAccessors from '../src/util/can-patch-native-accessors';
 
 describe('skatejs-named-slots', function () {
   it('version', function () {
     expect(version).to.be.a('string');
   });
 });
 
+if (canPatchNativeAccessors) {
```

Comment:
A better place to put this test might be in the `test/unit/shadow/polyfill.js` file as it seems to be specifically testing what happens to the original nodes when `attachShadow()` is called.